src/coreclr/jit/codegen.h

```diff
@@ -959,38 +959,35 @@ class CodeGen final : public CodeGenInterface
 #ifdef FEATURE_HW_INTRINSICS
     void genHWIntrinsic(GenTreeHWIntrinsic* node);
 #if defined(TARGET_XARCH)
-    void genHWIntrinsic_R_RM(GenTreeHWIntrinsic* node,
-                             instruction         ins,
-                             emitAttr            attr,
-                             regNumber           reg,
-                             GenTree*            rmOp,
-                             insOpts             instOptions = INS_OPTS_NONE);
-    void genHWIntrinsic_R_RM_I(GenTreeHWIntrinsic* node, instruction ins, emitAttr attr, int8_t ival);
+    void genHWIntrinsic_R_RM(
```

Comment:
same here? Have ` INS_OPTS_NONE` as default value?

Reply:
Same consideration. `genHWIntrinsic_*` very explicitly needs to ensure we're passing through any computed `insOpts`, otherwise it is very trivial to have computed the options and then missed passing them down in one place.